<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MtbUnisConfigWindow</class>
 <widget class="QDialog" name="MtbUnisConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bb_main">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Module name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_inputs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_inputs"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_outputs">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>271</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>Outputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_outputs"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="b_refresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_servos">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>261</width>
     <height>531</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_servos"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bb_main</sender>
   <signal>accepted()</signal>
   <receiver>MtbUnisConfigWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_main</sender>
   <signal>rejected()</signal>
   <receiver>MtbUnisConfigWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
